tensorboard/plugins/projector/tf_projector_plugin/BUILD

```diff
@@ -0,0 +1,36 @@
+package(default_visibility = ["//tensorboard:internal"])
+
+load("//tensorboard/defs:web.bzl", "tf_web_library")
+load("//tensorboard/defs:vulcanize.bzl", "tensorboard_html_binary")
+
+licenses(["notice"])  # Apache 2.0
+
+tf_web_library(
+    name = "projector_assets",
+    srcs = [
+        ":projector_binary.html",
+        "index.js"
+    ],
+    path = "/tf-projector",
+)
+
+tensorboard_html_binary(
+    name = "projector_binary",
+    input_path = "/tf-projector/tf-projector-plugin.html",
+    output_path = "/tf-projector/projector_binary.html",
+    deps = [
+        ":tf_projector_plugin",
+    ],
+)
+
+tf_web_library(
+    name = "tf_projector_plugin",
+    srcs = [
+        "tf-projector-plugin.html",
+    ],
+    path = "/tf-projector",
+    deps = [
+        "//tensorboard/plugins/projector/vz_projector",
+        "@com_google_fonts_roboto",
```

Comment:
To be clear, this bundles an extra copy of Roboto into the resulting
binary, right? (That’s an extra 644K raw and 492K compressed on a 3MB
Pip package.)

Also, if I’m reading this correctly, the Roboto font will still be
fetched from `/font-roboto/…`, which will resolve to the TensorBoard
root server, so we’re bundling the extra fonts and not actually using
them. :-(

Unfortunately, I don’t really see a quick fix for this with our current
build setup, given the whole “`fonts.bzl` generates a build file that
generates an html file with CSS that’s packaged into a `web_library`”
machinery. @nfelt and I discussed this briefly—there are a couple fixes
that we could take (such as making the generated CSS use relative paths
and serving the fonts from multiple paths with the same CSS), but I
can’t in good conscience ask for any of those in this PR. So… maybe open
an issue after this lands?

Reply:
Ok good news and bad news. 

> this bundles an extra copy of Roboto into the resulting

Good news:
It seems like this line adds a style tag that has content in this file: https://github.com/tensorflow/tensorboard/blob/052a8ea45e90804544d0291200ddea7ff85099f2/third_party/fonts.bzl#L400-L434

I tinkered around by with the fonts.bzl and basically omitted the style tag (and prevented CSS fetching the font) to check the difference in the file size. The CSS basically adds x-kB which wasn't too bad. 

So-so news:
It seems like, regardless, paper-style has dependency on roboto and it gets loaded almost no matter what. 
https://github.com/tensorflow/tensorboard/blob/052a8ea45e90804544d0291200ddea7ff85099f2/third_party/polymer.bzl#L1199-L1205

roboto.html -> typography.html -> paper-style.html

Bad news:
Yes, the font fetching fails when you use path_prefix flag. We can easily fix things by using a relative path in the fonts.bzl but it gets quite tricky for the dynamic plugins. We may be able to alleviate this by creating a dynamic plugin variant of the roboto.html which feels not good at the moment.

Reply:
Right: that was my analysis, too, so I think that we’re on the same page.